src/mode/node.rs

```diff
@@ -62,7 +62,7 @@ pub(crate) async fn start(args: Args) -> Result<()> {
     // TODO unwrap, better message
     let node_name = Uuid::new_v4();
     let node_name_str = node_name.as_hyphenated().to_string();
-    let node_attributes: HashMap<String, String> = Default::default(); //args.tag.into_iter().collect(); TODO
+    let node_attributes: HashMap<String, String> = args.tag.clone().into_iter().collect();
```

Comment:
Everything looks good to me. I don't have my head around the entire control server much to spot any issues.
In terms of the `BincodeJsonValue`, I'm not sure where that came from as I don't recall adding it, I mostly just copied from the axum implementation.

Here I noticed though, from a glance this `.clone()` might not be necessary.